harness/determined/pytorch/_pytorch_trial.py

```diff
@@ -674,85 +706,93 @@ def _save(self, path: pathlib.Path) -> workload.Response:
 class PyTorchTrial(det.Trial):
     """
     PyTorch trials are created by subclassing this abstract class.
-    """
 
-    # TODO(DET-3204): Add the following comments to the docstring.
-    # There are two different methods to define models, optimizers, and LR schedulers.
-    #
-    # 1. Define the abstract methods for the only model, optimizer, and LR scheduler and
-    #    Determine will initialize them for the users. Note this method doesn't support
-    #    multiple models, optimizers, and LR schedulers.
-    #
-    # 2. Initialize models, optimizers, and LR schedulers in the :meth:`__init__`.
-    #    If using this way, users should calculate the gradients with the losses and call an
-    #    optimization step for the optimizers on your own in :meth:`train_batch`.
-    #    Users are able to use arbitrary number of models, optimizers, and LR schedulers
-    #    and run the forward and backward passes and step optimizers in arbitrary orders.
+    We can do the following things in this trial class:
+
+    1. **Define models, optimizers, and LR schedulers**.
+
+       Initialize models, optimizers, and LR schedulers and wrap them with
+       ``wrap_model``, ``wrap_optimizer``, ``wrap_lrscheduler`` provided by
+       `PyTorchTrialContext <pytorch-trial-context_>`_ in the :meth:`__init__`.
+
+    2. **Run forward and backward passes**.
+
+       Call ``backward`` and ``step_optimizer`` provided by
+       `PyTorchTrialContext <pytorch-trial-context_>`_ in :meth:`train_batch`.
+       Note that we support arbitrary numbers of models, optimizers, and LR schedulers
+       and arbitrary orders of running forward and backward passes.
+
+    2. **Configure automatic mixed precision**.
```

Comment:
Numbering is wrong.  I'm ok using actual numbers instead of `#.` to make the python doctrings easier to read, but this is an example of why `#.` is a generally a good idea when docstring readability is not a concern.

As a sidenote, I think these should actually be bullet points and not numbered entries anyway.

Reply:
Changed it to bulleted list